indexd/default_settings.py

```diff
@@ -48,7 +49,17 @@
         "name": "DRS System",
         "host": "https://example.com/api/ga4gh/drs/v1/",
         "hints": [],
-        "type": "drs",
+        "type": {
```

Comment:
i'm not sure this is the best place to store this information 🤔 was there a feature doc about this? indexd's `dist` refers to external systems or other indexd instances. Why would it contain information about the current system? I think we need a new config to store this information?

Reply:
Ohhh i didnt realize thats what it was. I thought it was the current distribution information of indexd. I can add it to a different config. There was no feature doc on this. Was kind of a last miniute thing we needed for the ga4gh connect.